### PR TITLE
Fix typo test.cb in readme

### PR DESCRIPTION
**Pull request by forresst**

### CODE REVIEW

**Comment by jamestalmage** [comment]
:+1:  Thanks!

File: readme.md [hunk 1]
```diff
@@ -691,4 +691,3 @@ Concurrency is not parallelism. It enables parallelism. [Learn more.](http://sta
 	<br>
 	<br>
 </div>
-
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.
